### ISSUE TITLE
Q10 Needs Rewrite

### ISSUE THREAD

**Issue by steveafrost** [learn-co-curriculum/introduction-to-es6-quiz#3]
**Q10 What is the final value of array below?**
```
const array = [1, 2, 3]
const extension = [4, 5, 6]
 
array = [...extension, ...array]
```

**Answers**
```
[1, 2, 3]
[1, 2, 3, 4, 5, 6]
[4, 5, 6, 1, 2, 3]
```

I tried this out in node and you get a TypeError because the last line is trying to change a constant variable. Maybe it can be rewritten to ask what the value of newArray is and C would be the correct answer.
```
const array = [1, 2, 3]
const extension = [4, 5, 6]
 
newArray = [...extension, ...array]
```

Let me know your thoughts.

**Comment by achasveachas** [comment]
Agreed, unless the purpose of the question is to reinforce how `const`s can't be changed?

**Comment by bhabig** [comment]
I agree about rewriting. I think the point is to demonstrate that array will remain `[1,2,3]` but I think it would help to work in something like  'because a const can't be reassigned'  somehow without giving it away.
